### PR TITLE
Add course student

### PR DESCRIPTION
**Pull request by RemineralizedWater**
addCourse for student complete. Renamed addCourse for admin to createCourse. Also added number of small changes (file type fix, minor bugs, etc).

### CODE REVIEW

File: presentation/courses/addCourse.php [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <?php
+
```
Comment (smarkandu) [inline presentation/courses/addCourse.php:2]: I think "addCourse" should probably be placed in a new folder (ie. "student"), since it refers to a student.

**Comment by saladenicoise** [comment]
@RemineralizedWater Back button needed on Create Course Page and Add Course Page. No current way for the admin to get back to the main page.
Could be fixed if we add a dynamic nav bar that has: Home, Dashboard, Admin (Dynamically shows only if you are admin), Log Out/Log In (Dynamically show based on if logged in)

Cannot access addCourse if logged in as admin, intended?
- If so, change status message to something other than notL 

Classes for the css in printStat.js need to be changed to class="success" for those that are successful status messages

**Review by saladenicoise — COMMENTED** [review]
@RemineralizedWater Back button needed on Create Course Page and Add Course Page. No current way for the admin to get back to the main page.
Could be fixed if we add a dynamic nav bar that has: Home, Dashboard, Admin (Dynamically shows only if you are admin), Log Out/Log In (Dynamically show based on if logged in)

Cannot access addCourse if logged in as admin, intended?

    If so, change status message to something other than notL

Classes for the css in printStat.js need to be changed to class="success" for those that are successful status messages

File: presentation/courses/addCourse.php [hunk 2]
```diff
@@ -50,51 +51,22 @@
...
 <form method="post" action="../../data/courses/addCourseScript.php">
     <div>
-        <p>Course ID
-            <input name="course_code" type="text" size="10" required/><br/></p>
-        <p>Course Title
-            <input name="title" type="text" size="40" required/>
-        </p><br/>
+        <p>Student ID
+            <label>
+                <input name="id" type="number" size="10" placeholder="Student ID" required/>
```
Comment (smarkandu) [inline presentation/courses/addCourse.php:56]: Able to toggle integer values (strange) and also insert negative values as well (bad):

image

File: presentation/courses/addCourse.php [hunk 3]
```diff
@@ -50,51 +51,22 @@
...
+            </label><br/></p>
+        <p>Course Code
+            <label>
+                <input name="course_code" type="text" size="10" placeholder="Course Code" required/>
```
Comment (smarkandu) [inline presentation/courses/addCourse.php:60]: Course code should maybe be a dropdown or at least an autofill.

File: data/courses/addCourseScript.php [hunk 4]
```diff
@@ -8,49 +9,78 @@
...
 
+        $SQL = $conn->prepare("SELECT id FROM `registrar` WHERE id=?"); //Ensure max number of registered courses not reached
+        $SQL->bind_param('i', $id);
+        $SQL->execute();
+        $SQL->store_result();
+        $result = $SQL->num_rows;
+        $SQL->close();
+
+        // Business layer to check if max number of courses reached
+        require_once('../../business/courses/addCourseBusiness.php');
```
Comment (smarkandu) [inline data/courses/addCourseScript.php:66]: File doesn't exist.  Getting an error because of it when I try adding a course for a student.

**Review by smarkandu — CHANGES_REQUESTED** [review]
You're missing a file (ie. addCourseBusiness.php).  Please add it to your PR such that we can complete/continue the review.

File: data/courses/addCourseScript.php (hunk 4 above)
Comment (RemineralizedWater) [reply data/courses/addCourseScript.php:66]: Somehow it didn't push the new file I made, pushing fix soon.

Comment (RemineralizedWater) [reply data/courses/addCourseScript.php:66]: The commit didn't add the new file, pushing soon.

**Comment by RemineralizedWater** [comment]
> You're missing a file (ie. addCourseBusiness.php). Please add it to your PR such that we can complete/continue the review.

The commit didn't add the new file, pushing soon.

File: presentation/courses/addCourse.php (hunk 3 above)
Comment (RemineralizedWater) [reply presentation/courses/addCourse.php:60]: Optional features for later, but good idea.

File: presentation/courses/addCourse.php (hunk 2 above)
Comment (RemineralizedWater) [reply presentation/courses/addCourse.php:56]: Will check for negative integeters, but other suggestions are optional features for later, but good idea. Will push PR soon.

File: presentation/courses/addCourse.php (hunk 1 above)
Comment (RemineralizedWater) [reply presentation/courses/addCourse.php:2]: It could, but it also refers to a course. Perhaps modify later as optional feature.

**Comment by RemineralizedWater** [comment]
> 

Good idea for Back button, but for now will do workaround with having success/failure on add/create redirect to mainPage, then we can implement Back button (or even better, a nav header) later after main functionalities done.

Intended for admin not to be able to add course, as per project requirements that says Students can add courses, but doesn't say Admins can.

Will push change with "success" status as suggested.

**Comment by saladenicoise** [comment]
I meant for the class of the span to be changed or success not the url parameter

**Review by smarkandu — APPROVED** [review]

**Pull request merged**